tools/apigee/apigee.py

```diff
@@ -36,16 +37,23 @@
 # The apigee organization to migrate the keys to.
 # e.g. datcom-apigee
 _APIGEE_ORGANIZATION = os.environ.get("APIGEE_ORGANIZATION")
+# The URL to the Google Sheet with details of partners to be migrated.
+_SHEETS_URL = os.environ.get("SHEETS_URL")
+# The name of the worksheet at the SHEETS_URL with details of partners to be migrated.
+_WORKSHEET_NAME = os.environ.get("WORKSHEET_NAME")
```

Comment:
Maybe add asserts if these aren't set like we have for _DC_API_TARGET and _APIGEE_ORGANIZATION?

Also, consider adding a comment about expected sheet format (worksheet should have a project_id column)

Reply:
Done.